src/cve/data_models/config.py

```diff
@@ -72,6 +73,18 @@ def get_num_threads(cls, v: int) -> int:
 
 
 ########################### LLM Model Configs ###########################
+def prompt_file_check(prompt: str) -> str:
```

Comment:
Minor: it would be great for this to start with a verb/action for consistency other validators.

```suggestion
def check_prompt_file(prompt: str) -> str:
```